src/nuclio/projects/projects-functions-view/projects-functions-view.component.js

```diff
@@ -491,10 +491,16 @@
             // saves the name of sorted column
             ctrl.sortedColumnName = columnName;
 
-            if (lodash.isEmpty(ProjectsService.viewMode)) {
+            if (ProjectsService.viewMode !== 'projects') {
                 ctrl.functions = $filter('orderBy')(ctrl.functions, expression, ctrl.isReverseSorting);
             } else {
-                ctrl[ProjectsService.viewMode] = $filter('orderBy')(ctrl[ProjectsService.viewMode], expression, ctrl.isReverseSorting);
+                lodash.forEach(ctrl.projects, function (project) {
+                    var functions = lodash.get(project, 'ui.functions', []);
+
+                    functions = $filter('orderBy')(functions, expression, ctrl.isReverseSorting);
+
+                    lodash.set(project, 'ui.functions', functions);
```

Comment:
suggestion:
instead of using `lodash.get()` and `lodash.set` on the same object at the same path, you could use `lodash.update`

For example:
```js
lodash.update(project, 'ui.functions', function (functions) {
    return $filter('orderBy')(lodash.defaultTo(functions, []), expression, ctrl.isReverseSorting);
});
```